### PR TITLE
implement thematic break in the semantic HTML5 converter

### PR DESCRIPTION
**Pull request by redbow-kimee**

### CODE REVIEW

File: lib/asciidoctor/converter/semantic_html5.rb [hunk 1]
```diff
@@ -67,6 +67,10 @@ def convert_paragraph node
     end
   end
 
+  def convert_thematic_break node
+    "<hr />"
```
Comment (Mogztter) [inline lib/asciidoctor/converter/semantic_html5.rb:71]: I think we should use `<hr>` (at least for now).
In my opinion, we should open an issue to discuss if we want to support XHTML in the new converter or not.

Comment (redbow-kimee) [reply lib/asciidoctor/converter/semantic_html5.rb]: fixed with 0861630

**Comment by Mogztter** [comment]
I've created https://github.com/asciidoctor/asciidoctor/issues/4309 as a follow-up issue.

File: test/fixtures/semantic-html5-scenarios/thematic-break.adoc [hunk 2]
```diff
@@ -0,0 +1,5 @@
+this is a first paragraph.
```
Comment (Mogztter) [inline test/fixtures/semantic-html5-scenarios/thematic-break.adoc:1]:
```suggestion
This is a first paragraph.
```

File: test/fixtures/semantic-html5-scenarios/thematic-break.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<p>
+this is a first paragraph.
```
Comment (Mogztter) [inline test/fixtures/semantic-html5-scenarios/thematic-break.html:2]:
```suggestion
This is a first paragraph.
```

**Pull request merged**